src/Compilers/CSharp/Test/Syntax/LexicalAndXml/PreprocessorTests.cs

```diff
@@ -3433,22 +3439,6 @@ public void TestPragmaWarningDisable()
             });
         }
 
-        [Fact]
-        [Trait("Feature", "Directives")]
-        public void TestPragmaWarningDisableNullable()
-        {
-            var text = @"#pragma warning disable nullable";
```

Comment:
>#pragma warning disable nullable [](start = 25, length = 32)

It may be good to keep a nullable test with `#pragma warning disable nullable` and variants, showing that it has no effect (it is silently ignored).

Reply:
Could you bring a `#pragma warning disable nullable` test back in `NullableReferenceTypeTests` as well, to show that it has no effect (it is silently ignored)?

---
In reply to: [295031373](https://github.com/dotnet/roslyn/pull/36464#discussion_r295031373) [](ancestors = 295031373)

Reply:
Added test `PragmaNullable_NoEffect`

---
In reply to: [295035967](https://github.com/dotnet/roslyn/pull/36464#discussion_r295035967) [](ancestors = 295035967,295031373)